mobsos-data-processing/src/main/java/i5/las2peer/services/mobsos/dataProcessing/MobSOSDataProcessingService.java

```diff
@@ -54,6 +54,7 @@ public class MobSOSDataProcessingService extends Service {
 	private Set<String> triggerFunctions = new HashSet<String>();
 	private ArrayList<BotMessage> botMessages = new ArrayList<BotMessage>();
 	private ArrayList<String> xAPIstatements = new ArrayList<String>();
+	//private boolean sendStatementsToBots = true; // True if xAPI statements should also be sent to the social bot manager service
```

Comment:
why is this commented out?

Reply:
I am not sure. This was introduced in https://github.com/rwth-acis/mobsos-data-processing/commit/1bcbd2d9f57afc8d25c9763ecbd3cce56bbba1eb